zip-0316.rst

```diff
@@ -486,14 +495,29 @@ should be sufficient because experiments are expected to be reasonably
 short-term, and should otherwise be either standardized in a ZIP (and
 allocated a Typecode outside this reserved range) or discontinued.
 
+New types SHOULD maintain the same distinction between FVK and IVK
+authority as existing types, i.e. an FVK is intended to give access to
+view all transactions to and from the address, while an IVK is intended
+to give access only to view incoming payments (as opposed to change).
+
 
 Deriving a UIVK from a UFVK
 ---------------------------
 
-Each UIVK Encoding can straightforwardly be derived from the
-corresponding UFVK Encoding, without changing the Typecode. In the
-case of a Transparent P2PKH UFVK Encoding, the UIVK Encoding is the
-same.
+The following derivations are applied to each component FVK:
+
+* For a Sapling FVK, the corresponding Sapling IVK is obtained as
+  specified in [#protocol-saplingkeycomponents]_.
+
+* For an Orchard FVK, the corresponding Orchard IVK is obtained as
+  specified in [#protocol-orchardkeycomponents]_.
+
+* For a Transparent P2PKH FVK, the corresponding Transparent P2PKH IVK
+  is obtained from the extended public key corresponding to the account's
+  non-change key path, i.e. by deriving the child key with non-hardened
+  index :math:`0`. It is encoded in the same way as the FVK Encoding.
```

Comment:
This is no longer quite correct, since the FVK doesn't contain the extended public key; this should probably refer instead to the function `CKDpub((Kpar, cpar), i) → (Ki, ci)` as defined in BIP 32.